modules/core/include/opencv2/core/version.hpp

```diff
@@ -8,7 +8,7 @@
 #define CV_VERSION_MAJOR    5
 #define CV_VERSION_MINOR    0
 #define CV_VERSION_REVISION 0
-#define CV_VERSION_STATUS   "alpha-pre"
+#define CV_VERSION_STATUS   "alpha"
```

Comment:
must be `-alpha`

It could be easily validated if you add reference on PR with any previous releases.

https://github.com/opencv/opencv/commit/71d3237a093b60a27601c20e9ee6c3e52154e8b1